ADR/ADR009-replatforming-to-ECS.md

```diff
@@ -0,0 +1,38 @@
+# ADR009: Re-platforming GOV.UK Forms to AWS ECS Fargate
+
+Date: 2022-10-27
+
+## Status
+
+Accepted
+
+## Context
+
+GOV.UK Forms currently runs on GOV.UK PaaS which is being decommissioned in
```

Comment:
How about a link to the "Why we’ve decided to decommission GOV.UK PaaS" blog post from [GOV.UK PaaS which is being decommissioned in December 2023](https://gds.blog.gov.uk/2022/07/12/why-weve-decided-to-decommission-gov-uk-paas-platform-as-a-service/)